techeria_controller/techeria_app/models.py

```diff
@@ -182,62 +182,44 @@ class ShippingAddress(models.Model):
 
     def __str__(self):
 	    return self.address
-    
+
     class Meta:
         db_table = "shippingaddress"
-        
-# class Order(models.Model):
-#     STATUS = (
-#         ('New', 'New'),
-#         ('Accepted', 'Accepted'),
-#         ('Completed', 'Completed'),
-#         ('Cancelled', 'Cancelled'),
-#     )
-
-#     user = models.ForeignKey(BuyerModel, on_delete=models.SET_NULL, null=True)
-#     payment = models.ForeignKey(Payment, on_delete=models.SET_NULL, blank=True, null=True)
-#     order_number = models.CharField(max_length=20)
-#     first_name = models.CharField(max_length=50)
-#     last_name = models.CharField(max_length=50)
-#     phone = models.CharField(max_length=15)
-#     email = models.EmailField()
-#     mobile_number = models.CharField(max_length=255)
-#     address = models.CharField(max_length=255)
-#     city = models.CharField(max_length=255)
-#     state = models.CharField(max_length=255)
-#     zip_code = models.CharField(max_length=255)
-#     country = models.CharField(max_length=20)
-#     order_note = models.CharField(max_length=100, blank=True)
-#     order_total = models.FloatField()
-#     tax = models.FloatField()
-#     status = models.CharField(max_length=10, choices=STATUS, default='New')
-#     ip = models.CharField(blank=True, max_length=20)
-#     is_ordered = models.BooleanField(default=False)
-#     created_at = models.DateTimeField(auto_now_add=True)
-#     updated_at = models.DateTimeField(auto_now=True)
-
-
-#     def full_name(self):
-#         return f'{self.first_name} {self.last_name}'
-
-#     def full_address(self):
-#         return f'{self.address_line_1} {self.address_line_2}'
-
-#     def __str__(self):
-#         return self.first_name
-
-
-# class OrderProduct(models.Model):
-#     order = models.ForeignKey(Order, on_delete=models.CASCADE)
-#     payment = models.ForeignKey(Payment, on_delete=models.SET_NULL, blank=True, null=True)
-#     user = models.ForeignKey(BuyerModel, on_delete=models.CASCADE)
-#     product = models.ForeignKey(Products, on_delete=models.CASCADE)
-#     quantity = models.IntegerField()
-#     product_price = models.FloatField()
-#     ordered = models.BooleanField(default=False)
-#     created_at = models.DateTimeField(auto_now_add=True)
-#     updated_at = models.DateTimeField(auto_now=True)
-
-#     def __str__(self):
-#         return self.product.product_name
 
+
+
+class PlaceOrder(models.Model):
```

Comment:
This should add an order to the orders database table, but it doesn't happen so we should take a closer look at this and make sure it works.